src/main/java/org/apache/ofbiz/ofbiztutorial/usermgr/services/UserMgrServices.java

```diff
@@ -0,0 +1,164 @@
+package org.apache.ofbiz.ofbiztutorial.usermgr.services;
+
+
+import org.apache.ofbiz.base.util.Debug;
+import org.apache.ofbiz.base.util.UtilMisc;
+import org.apache.ofbiz.entity.GenericValue;
+import org.apache.ofbiz.service.DispatchContext;
+import org.apache.ofbiz.service.GenericServiceException;
+import org.apache.ofbiz.service.ServiceUtil;
+<<<<<<< HEAD
+import org.apache.ofbiz.base.util.UtilDateTime;
+
+=======
+>>>>>>> b5bdf4e25b3ebee19ea8ad8ad27302fa32a4f9f6
+
+import java.util.Map;
+
+public class UserMgrServices{
+    public static final String MODULE = UserMgrServices.class.getName();
+
+    public static Map<String, Object> createUserByJavaService(DispatchContext dctx, Map<String, ? extends Object> context){
+        Map<String, Object> result = ServiceUtil.returnSuccess();
+        String userLoginId = (String) context.get("userLoginId");
+        String password = (String) context.get("password");
+        String confirmPassword = (String) context.get("confirmPassword");
+        String firstName = (String) context.get("firstName");
+        String lastName = (String) context.get("lastName");
+        String middleName = (String) context.get("middleName");
+//        String birthDate = (String) context.get("birthDate");
+        String address1 = (String) context.get("address1");
+        String address2 = (String) context.get("address2");
+        String city = (String) context.get("city");
+        String postalCode = (String) context.get("postalCode");
+        String countryGeoId = (String) context.get("countryGeoId");
+        String countryCode = (String) context.get("countryCode");
+        String areaCode = (String) context.get("areaCode");
+        String contactNumber = (String) context.get("contactNumber");
+        String emailAddress = (String) context.get("emailAddress");
+        GenericValue userLogin = (GenericValue) context.get("userLogin");
+
+        System.out.println("============="+userLogin+"===============");
+        // Check if passwords match
+        if (!password.equals(confirmPassword)) {
+            return ServiceUtil.returnError("Passwords do not match.");
+        }
+
+        // Invoke createPersonAndUserLogin service to create user and associated person
+        Map<String, Object> createPersonParams = Map.of(
```

Comment:
use Ofbiz method instead of Map.of

UtilMisc.toMap

Reply:
Sure sir I'll implement the change in the next commit.